omnibus/partybus/lib/partybus/dsl_runner.rb

```diff
@@ -10,19 +10,53 @@ def initialize(migration_file)
     @file_path     = migration_file.path
   end
 
-  def run
+  def load
```

Comment:
[nit] I'm generally not a fan of overloading methods from `Kernel`, but this seems okay.

Reply:
I can change it to `load_migration`.  That'll also fix the syntax highlighting :D